test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -1,677 +1,774 @@
 module UnitTest {
```

Comment:
How about a simple top-level doc string above `module UnitTest`?

Reply:
I  think it good to have a top level doc string. I have added that and also added 
```chapel
pragma "no doc"
``` 
where necessary.

Reply:
OK, can keep it simple for now, like:

```chpl
/* Prototype UnitTest module */
module UnitTest {
 ...
}
```

Reply:
Thanks. Made necessary changes.